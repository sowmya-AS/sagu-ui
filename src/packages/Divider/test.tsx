import React from 'react'
import { render } from '../../utils/testUtils'

import Divider from '.'

describe('<Divider />', () => {
  it('should render Divider correctly', () => {
    const { container } = render(<Divider />)
    expect(container).toMatchSnapshot()
  })
})
