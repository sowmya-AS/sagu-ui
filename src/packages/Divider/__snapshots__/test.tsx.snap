// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should render Divider correctly 1`] = `
.c0 {
  height: 0.01rem;
  background: #808080;
  border: 0;
}

<div>
  <hr
    class="c0"
  />
</div>
`;
